examples/DrawX/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+
+// Information about this assembly is defined by the following attributes. 
+// Change them to the values specific to your project.
+
+[assembly: AssemblyTitle("DrawX")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("")]
+[assembly: AssemblyCopyright("andydent")]
```

Comment:
Yes, it's a public sample, sorry I missed this file as I was driving it from the other one DrawXnative.